Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:10.18.1
```

Comment:
Why this node version?

Reply:
Because that's the version I'm running here, so that what I'm running all the tests against.